subset/cloud/test_udmi

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+
+REPORT=/tmp/report.txt
+
+# Necessary to reach gcp. Should be done by framework but this works for now.
+route add default gw $GATEWAY_IP
+
+gcp_cred=/config/inst/gcp_service_account.json
+gcp_topic=target
```

Comment:
@grafnu The gcp_topic is hardcoded here to 'target', but the Pubber.java seems to publish its test pointset messages to events/pointset. Is that right?

see line 32 of https://github.com/grafnu/daq/blob/0014ec03d4da1ba5dd5b16f209f6dd61bc968ac7/pubber/src/main/java/daq/pubber/Pubber.java 

line 232 of the same file for where the test message is being sent